web/Controllers/PropertyController.cs

```diff
@@ -140,6 +140,32 @@ public ActionResult AddNewProperty_Cancel()
         // Go to View Properties page
         return RedirectToAction("ViewProperties", "Landlord");
     }
+    
+    [HttpPost]
+    [Authorize(Roles = "Landlord")]
+    [Authorize(Roles = "Admin")]

```

Comment:
I think stacking Authorize attributes does an AND, so you have to be both a landlord and an admin - I think you instead want `[Authorize(Roles="Landlord,Admin")]`, but do check!